pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -33,6 +33,11 @@ const (
 	CertificateSourceTypeSecret CertificateSourceType = "Secret"
 	// CertificateSourceTypeSecretLowCase certificate coming from a user provided secret
 	CertificateSourceTypeSecretLowCase CertificateSourceType = "secret"
+
+	// CertificateSourceTypeNoneNoEncryption no certificate encryption disabled
+	CertificateSourceTypeNoneNoEncryption CertificateSourceType = "None"
```

Comment:
@rigazilla I think we don't need lowercase for the new parameters. Both stiles was for migration case only

Reply:
@dmvolod great! I've forgot about it